### DISCUSSION TITLE
GitLab platform ignores `issues_access_level`

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#26533] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

v37.102.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

We have projects that have their `issues_access_level` setting set to `"disabled"`. As a result, it is expected behavior that no Dependency Dashboards can be created, and there is no use for the GitLab platform module in attempting it still. Doing so triggers:

```
 WARN: Could not ensure issue (repository=Discovery/DeliveryPipeline/Jenkins/Jobs)
       "err": {
         "name": "HTTPError",
         "code": "ERR_NON_2XX_3XX_RESPONSE",
         "timings": {
           "start": 1704506076863,
           "socket": 1704506076863,
           "lookup": 1704506076867,
           "connect": 1704506076869,
           "secureConnect": 1704506076871,
           "upload": 1704506076871,
           "response": 1704506076937,
           "end": 1704506076938,
           "phases": {
             "wait": 0,
             "dns": 4,
             "tcp": 2,
             "tls": 2,
             "request": 0,
             "firstByte": 66,
             "download": 1,
             "total": 75
           }
         },
         "message": "Response code 403 (Forbidden)",
         "stack": "HTTPError: Response code 403 (Forbidden)\n    at Request.<anonymous> (/opt/containerbase/tools/renovate/37.102.0/node_modules/got/dist/source/as-promise/index.js:118:42)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)",
...
```

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by viceice** [comment]
it's not a problem, it needs someone to implement it like we do on other platforms like gitea, where we detect disabled issues.
